base/utf8proc.jl

```diff
@@ -121,6 +121,12 @@ end
 
 charwidth(c::Char) = Int(ccall(:utf8proc_charwidth, Cint, (UInt32,), c))
 
+# faster x+y that does no overflow checking
+p(x::Char, y::UInt32) = reinterpret(Char, reinterpret(UInt32, x) + y)
```

Comment:
yep... I agree with @tkelman, no 1-letter (or 1 letterlike number such as 𝟙 :grinning:!) function names...
Also... this function wouldn't have been necessary if you all had agreed to my PR #11103! :grinning:  Maybe you could reopen #11103, and make this a lot cleaner?